app/models/searchgov_url.rb

```diff
@@ -34,14 +37,18 @@ class SearchgovUrl < ActiveRecord::Base
     column_name: proc {|url| !url.fetched? ? 'unfetched_urls_count' : nil },
     column_names: { ['searchgov_urls.last_crawled_at IS NULL'] => 'unfetched_urls_count' }
 
-  scope :fetch_required, -> { where('last_crawled_at IS NULL OR lastmod > last_crawled_at') }
+  scope :fetch_required, ->() do
+    where('last_crawled_at IS NULL OR
```

Comment:
minor nit, but this would read a bit better if the conjunctions preceded the conditions:
``` ruby
where('last_crawled_at IS NULL
       OR lastmod > last_crawled_at
       OR enqueued_for_reindex')
```